package org.example.demo.model;

import java.time.LocalDateTime;

public class Log {
    private Long id;
    private String operator;
    private String type;
    private LocalDateTime time;

    public Long getId() { return id; }
    public void setId(Long id) { this.id = id; }

    public String getOperator() { return operator; }
    public void setOperator(String operator) { this.operator = operator; }

    public String getType() { return type; }
    public void setType(String type) { this.type = type; }

    public LocalDateTime getTime() { return time; }
    public void setTime(LocalDateTime time) { this.time = time; }
}